code/sorting/src/bubble_sort/bubble_sort.cpp

```diff
@@ -1,50 +1,42 @@
-/*
- * Part of Cosmos by OpenGenus Foundation
- *
- * bubble sort synopsis
- *
- * template<typename _Bidirectional_Iter, typename _Compare>
- * void
- * bubbleSort(_Bidirectional_Iter begin, _Bidirectional_Iter end, _Compare compare);
- *
- * template<typename _Bidirectional_Iter>
- * void
- * bubbleSort(_Bidirectional_Iter begin, _Bidirectional_Iter end);
- */
 
-#include <functional>
-
-template<typename _Bidirectional_Iter, typename _Compare>
-void
-bubbleSort(_Bidirectional_Iter begin, _Bidirectional_Iter end, _Compare compare)
-{
-    if (begin != end)
+#include <iostream>
+using namespace std;
+int recbub(int arr[],int n)
+{   
+   int i,temp;
+   if(n==1)
+   {
+   return n;
+   }
+    else
```

Comment:
Why did you remove the old code?

Reply:
@arnavb I write bubble sort code using recursion, that's why I remove the old one .
or Can I make another file for this ?

Reply:
@arnavb Please view my other pull requests also.